package.json

```diff
@@ -26,34 +26,41 @@
     "react-dom": "^16.8.6"
   },
   "scripts": {
-    "start": "cd demo-app && react-scripts-ts start",
+    "start": "webpack-dev-server --mode development",
     "test": "react-scripts-ts test --env=jsdom",
     "coverage": "react-scripts-ts test --mapCoverage --env=jsdom --coverage",
     "build": "webpack -p",
     "docs": "typedoc --out docs --name 'React Wodden Tree' --target es6 --excludePrivate --excludeProtected --excludeExternals --mode file src/**/{Tree,types}.tsx"
   },
   "devDependencies": {
+    "@babel/core": "^7.4.4",
+    "@babel/preset-env": "^7.4.4",
+    "@babel/preset-react": "^7.0.0",
     "@types/jest": "^24.0.11",
     "@types/node": "^11.13.4",
-    "@types/react": "^16.8.13",
+    "@types/react": "^16.8.17",
     "@types/react-dom": "^16.8.4",
     "@types/react-redux": "^7.0.6",
     "@types/react-test-renderer": "^16.8.1",
-    "awesome-typescript-loader": "^5.2.1",
+    "babel-loader": "^8.0.6",
     "copy-webpack-plugin": "^4.6.0",
     "css-loader": "^2.1.1",
+    "file-loader": "^3.0.1",
     "font-awesome": "^4.7.0",
     "react-redux": "^7.0.3",
     "react-scripts-ts": "^3.1.0",
     "react-test-renderer": "^16.8.6",
     "react-wooden-tree": "file:./",
     "redux": "^4.0.1",
     "serve": "^11.0.0",
+    "style-loader": "^0.23.1",
+    "ts-loader": "^6.0.0",
     "tslint-loader": "^3.6.0",
     "typedoc": "^0.14.2",
     "typescript": "^3.4.5",
-    "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.2"
+    "webpack": "^4.31.0",
```

Comment:
In general I'm not sure it is a great idea to upgrade package versions in this PR. When you're doing something that's not dependent on version upgrades, you should not upgrade. The amount of changes should be always lowest as possible.